boefjes/boefjes/config.py

```diff
@@ -76,33 +79,38 @@ class Settings(BaseSettings):
         8000,
         description="Host port of the Boefje API server",
     )
-
     boefje_docker_network: str = Field(
         "bridge",
         description="Docker network to run Boefjes in",
         env="BOEFJE_DOCKER_NETWORK",
     )
-
-    bytes_api: AnyHttpUrl = Field(..., example="http://localhost:8002", description="Bytes API URL", env="BYTES_API")
-    bytes_username: str = Field(..., example="test", description="Bytes JWT login username", env="BYTES_USERNAME")
-    bytes_password: str = Field(..., example="secret", description="Bytes JWT login password", env="BYTES_PASSWORD")
+    bytes_api: AnyHttpUrl = Field(
+        ..., examples=["http://localhost:8002"], description="Bytes API URL", validation_alias="BYTES_API"
+    )
+    bytes_username: str = Field(
+        ..., examples=["test"], description="Bytes JWT login username", validation_alias="BYTES_USERNAME"
+    )
+    bytes_password: str = Field(
+        ..., examples=["secret"], description="Bytes JWT login password", validation_alias="BYTES_PASSWORD"
+    )
 
     span_export_grpc_endpoint: Optional[AnyHttpUrl] = Field(
-        None, description="OpenTelemetry endpoint", env="SPAN_EXPORT_GRPC_ENDPOINT"
+        None, description="OpenTelemetry endpoint", validation_alias="SPAN_EXPORT_GRPC_ENDPOINT"
     )
 
-    class Config:
-        env_prefix = "BOEFJES_"
-
-        @classmethod
-        def customise_sources(
-            cls,
-            init_settings: SettingsSourceCallable,
-            env_settings: SettingsSourceCallable,
-            file_secret_settings: SettingsSourceCallable,
-        ) -> Tuple[SettingsSourceCallable, ...]:
-            backwards_compatible_settings = BackwardsCompatibleEnvSettings()
-            return env_settings, init_settings, file_secret_settings, backwards_compatible_settings
+    model_config = SettingsConfigDict(env_prefix="BOEFJES_")
+
+    @classmethod
+    def settings_customise_sources(
+        cls,
+        settings_cls: Type[BaseSettings],
+        init_settings: PydanticBaseSettingsSource,
+        env_settings: PydanticBaseSettingsSource,
+        dotenv_settings: PydanticBaseSettingsSource,
+        file_secret_settings: PydanticBaseSettingsSource,
+    ) -> Tuple[PydanticBaseSettingsSource, ...]:
+        backwards_compatible_settings = BackwardsCompatibleEnvSettings(settings_cls)
+        return env_settings, init_settings, file_secret_settings, backwards_compatible_settings
```

Comment:
Dotenv_settings is missing and the order is changed.

```suggestion
        return init_settings, env_settings,  dot_env_settings, file_secret_settings, backwards_compatible_settings
```